Design.md

```diff
@@ -0,0 +1,96 @@
+# Mosaic Design
+## General Concept
+There are two key steps for preparing OPERA DISP data for presentation in the MapBox Tiling Service:
+
+1. The discrete, non-rectangular, and rotated frames of OPERA DISP products need to be summarized into a continuous single-layer raster representation.
+2. This continuous representation needs to delivered to MapBox in a format that is simple for it to ingest.
+
+To meet the first requirement, we take discrete OPERA DISP products and transform them into a set of square 1x1 degree COGs. This strategy has worked well for other projects (such as the Global Seasonal Coherence dataset) and allows us to easily convert the data to the tileset representation that MapBox. It also provides us with a great starting point to provide this data to users via other avenues (GIBS, ESRI Image Server, TMS, etc.) in the future. There are intricacies for mosaicking each product, which are detailed in other sections, but in general each tile is created separately using 1-4 input OPERA DISP products. Products are overlaid so that products along the same absolute orbit are continuous, and neighboring relative orbit data are overlaid from east to west. The 1x1 degree COGs maintain the same datatype and data values as the original products.
```

Comment:
Generating the 1x1 data set sounds like an opportunity rather than a requirement, yes? Given the time constraints of the project, we could focus on building the "10 or fewer tiled GeoTiffs with a UInt8 datatype", rather than adding the requirement of the intermediate 1x1 COG data set.